libs/api/domains/vehicles/src/lib/api-domains-vehicles-mileage.resolver.ts

```diff
@@ -101,4 +101,16 @@ export class VehiclesMileageResolver {
       permno: overview?.permno ?? '',
     })
   }
+
+  @ResolveField('canUserRegisterVehicleMileage', () => Boolean, {
+    nullable: true,
+  })
+  resolveCanUserRegisterMileage(
+    @Context('req') { user }: { user: User },
+    @Parent() overview: VehicleMileageOverview,
+  ): Promise<boolean> {
```

Comment:
Should this be a complex type that has `canRegister` as property? Then if we need to add more info about the mileage (e.g. if somebody should not see it? dunno, but something might come up!) we could add property instead of having to create a new thing

Reply:
nm, reading it wrong